run_dir/static/js/proj_meta_compare.js

```diff
@@ -1,604 +1,619 @@
+        });
 
-        var sect1 = $('#proj_meta_yvalue option:selected').data('section');
-        var key1 = $('#proj_meta_yvalue').val();
-        var sect2 = $('#proj_meta_xvalue option:selected').data('section');
-        var key2 = $('#proj_meta_xvalue').val();
-        var sect3 = $('#proj_meta_colvalue option:selected').data('section');
-        var key3 = $('#proj_meta_colvalue').val();
-
-        if(key1 != '' && key2 != ''){
-            plot_meta({
-                'y': [sect1, key1],
-                'x': [sect2, key2],
-                'color': [sect3, key3],
-            });
-        }
-    });
+        // Copy to clipboard button
+        new Clipboard('#projMeta_copyRaw', {
+            text: function(trigger) {
+                return JSON.stringify(project_data, null, "  ");
+            }
+        });
 
-    // Download all data button
-    $('#projMeta_downloadAll').click(function(e){
-        e.preventDefault();
-        pmeta_download();
-    });
+        // Change between log and linear axes
+        $('.meta_xLogLin_buttons button').click(function(e){
+            e.preventDefault();
+            $('.meta_xLogLin_buttons button').removeClass('active');
+            $(this).addClass('active');
+            if($(this).val() == 'log'){ xLogAxis = 'logarithmic'; }
+            else { xLogAxis = 'linear'; }
+            $('#proj_meta_plot').highcharts().xAxis[0].update({ type: xLogAxis});
+        });
+        $('.meta_yLogLin_buttons button').click(function(e){
+            e.preventDefault();
+            $('.meta_yLogLin_buttons button').removeClass('active');
+            $(this).addClass('active');
+            if($(this).val() == 'log'){ yLogAxis = 'logarithmic'; }
+            else { yLogAxis = 'linear'; }
+            $('#proj_meta_plot').highcharts().yAxis[0].update({ type: yLogAxis});
+        });
 
-    // Copy to clipboard button
-    new Clipboard('#projMeta_copyRaw', {
-        text: function(trigger) {
-            return JSON.stringify(project_data, null, "  ");
-        }
     });
 
-    // Change between log and linear axes
-    $('.meta_xLogLin_buttons button').click(function(e){
-        e.preventDefault();
-        $('.meta_xLogLin_buttons button').removeClass('active');
-        $(this).addClass('active');
-        if($(this).val() == 'log'){ xLogAxis = 'logarithmic'; }
-        else { xLogAxis = 'linear'; }
-        $('#proj_meta_plot').highcharts().xAxis[0].update({ type: xLogAxis});
-    });
-    $('.meta_yLogLin_buttons button').click(function(e){
-        e.preventDefault();
-        $('.meta_yLogLin_buttons button').removeClass('active');
-        $(this).addClass('active');
-        if($(this).val() == 'log'){ yLogAxis = 'logarithmic'; }
-        else { yLogAxis = 'linear'; }
-        $('#proj_meta_plot').highcharts().yAxis[0].update({ type: yLogAxis});
+    //Kind of works, flashes a bit
+    $(document).click(function() {
+        $('#projects_meta_input').val('');
     });
```

Comment:
```suggestion
```
Instead of this, you could add the below after line 70
` $("#projects_meta_input").typeahead("val", "");`
 and also remove line 60 (`$('#projects_meta_input').val('');`)